test/quantum/run_strategy_test.exs

```diff
@@ -13,19 +13,19 @@ defmodule Quantum.RunStrategyTest do
   test "run strategy local" do
     job = Job.new(Scheduler.config(run_strategy: Quantum.RunStrategy.Local))
     assert %Job{} = job
-    assert [_] = NodeList.nodes(job.run_strategy, job)
+    assert [:nonode@nohost] == NodeList.nodes(job.run_strategy, job)
```

Comment:
Please use `Node.self/0` instead of a fixed atom. This may fail depending on the configuration.